app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+
+    def __init__(self):
+        self._connection = sqlite3.connect('cinema.sqlite')
+        self.table_name = 'actors'
+
+    def create(self, first_name: str, last_name: str) -> None:
+
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (format) VALUES (?)",
+            (first_name, last_name,)

```

Comment:
The SQL query in the `create` method is incorrect. The placeholder `format` should be replaced with the actual column names `first_name` and `last_name`. The query should look like: `INSERT INTO {self.table_name} (first_name, last_name) VALUES (?, ?)`.